logging/zerolog/ctxzr/context.go

```diff
@@ -0,0 +1,80 @@
+package ctxzr
+
+import (
+	"context"
+	grpc_ctxtags "github.com/grpc-ecosystem/go-grpc-middleware/tags"
+	"github.com/rs/zerolog"
+)
+
+type ctxMarker struct{}
+
+type CtxLogger struct {
+	Logger *zerolog.Logger
+	Fields map[string]interface{}
+}
```

Comment:
Is there anyway that we dont have to use our own CtxLogger? And instead add the fields onto the zerolog.Logger?